src/zycast.c

```diff
@@ -0,0 +1,336 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * zycast - push images via multicast to a ZyXEL bootloader
+ *
+ * Many ZyXEL devices supports image manipulation using a multicast
```

Comment:
Can you list (some of) the known supported devices? Maybe in the sources, otherwise definitely in the commit message. At least to know which devices you tested this on.